src/ExternalACL.h

```diff
@@ -58,7 +39,13 @@ class ACLExternal : public ACL
     bool valid () const override;
     bool empty () const override;
 
-protected:
+private:
+    static ACLChecklist::AsyncStarter StartLookup;
```

Comment:
This static function member declaration is correct but _looks_ like a (banned by the "no new globals" rule) static data member declaration. Is there a form of this declaration that would tell the reader that StartLookup is a function (without duplicating/spelling-out AsyncStarted profile, of course)? The following does _not_ work, right?

```C++
static ACLChecklist::AsyncStarter StartLookup();
```

Reply:
I think we are ready for this final step.

Reply:
Done (2f7556b).